string name
---
string name
string cad
string description
int16 number_of_tasks
float64 last_saved
uint8 state
string state_description
